x-pack/plugins/security_solution/public/common/hooks/use_upgrade_security_packages.ts

```diff
@@ -35,25 +30,9 @@ const sendUpgradeSecurityPackages = async (
   });
 };
 
-/**
- * Checks with the ingest manager if the current user making these requests has the right permissions
- * to install the endpoint package.
- *
- * @param http an http client for sending the request
- * @param options an object containing options for the request
- */
-const sendCheckPermissions = async (
```

Comment:
We don't need this anymore because the `useEndpointPrivileges()` now calls the Fleet API to check permissions